packages/synapse-react-client/src/lib/containers/oauth/CreateOAuthClient.tsx

```diff
@@ -199,246 +214,224 @@ export const CreateOAuthModal: React.FunctionComponent<
-                limitations.
-              </Typography>
-
-              {isEdit && (
-                <Typography style={{ marginTop: '16px' }} variant="label">
-                  Client ID: {client?.client_id}
-                </Typography>
-              )}
-              <Row>
-                <Col lg={6} md={6} sm={12} xs={12}>
-                  <Form.Group className="required">
-                    <Form.Label htmlFor="clientName">Client Name</Form.Label>
-                    <Form.Control
-                      required
-                      onChange={e => setClientName(e.target.value)}
-                      placeholder="Client Name"
-                      type="text"
-                      value={clientName}
-                      id="clientName"
-                    />
-                  </Form.Group>
-                </Col>
-                <Col lg={6} md={6} sm={12} xs={12}>
-                  <Form.Label className="required" htmlFor="clientUri">
-                    Client Homepage
-                  </Form.Label>
-                  <Form.Control
-                    onChange={e => setClientUri(e.target.value)}
-                    placeholder="https://"
-                    type="text"
-                    value={clientUri}
-                    id="clientUri"
-                  />
-                </Col>
-              </Row>
-              <Row>
-                {!isEdit && (
-                  <>
-                    <Col lg={6} md={6} sm={12} xs={12}>
-                      <Form.Group className="required">
-                        <Form.Label htmlFor="redirect-uri-0">
-                          Redirect URI(s)
-                        </Form.Label>
-                        <Tooltip title={uriHelpMessage} placement="top">
-                          <HelpOutlineTwoTone className={`HelpButton`} />
-                        </Tooltip>
-                        {redirectUris?.map((singleUri, idx) => (
-                          <div key={idx}>
-                            <Form.Control
-                              name="uri"
-                              required
-                              id={`redirect-uri-${idx}`}
-                              onChange={e => handleUriChange(e, idx)}
-                              value={singleUri.uri}
-                              placeholder="https://"
-                              type="text"
-                            />
-                            {redirectUris.length > 1 && (
-                              <button
-                                onClick={() => handleRedirectUriRemove(idx)}
-                              >
-                                <IconSvg
-                                  icon="delete"
-                                  sx={{ color: 'error.main' }}
-                                />
-                              </button>
-                            )}
+        Sector Identifier URI
+      </InputLabel>
+      <MuiTextField
```

Comment:
SRC TextField changes the label to the primary color on focus. Use MUI TextField instead, so that the error label color persists on focus when editing an existing client

Reply:
I'd prefer we use the MUI versions when possible, so this is fine. There's just something strange with our styling/implementation. For example MUI's [TextField](https://mui.com/material-ui/react-text-field/) internally handles adding an InputLabel, so we shouldn't need to provide one.

Reply:
Oh good call, I missed that! I updated the inputs to use MUI's TextField wrapped in a StyledFormControl instead

Reply:
Tracking that set of work here: https://sagebionetworks.jira.com/browse/PORTALS-2658